src/dev-app/theme.scss

```diff
@@ -13,6 +14,9 @@
 @include angular-material-typography-mdc();
 @include mat-edit-typography(mat-typography-config());
 
+// Include base styles for strong focus indicators.
+@include mat-strong-focus-indicators();
```

Comment:
For a follow-up PR: I'd like to make this toggle-able in the dev app via a button in the top right (like RTL and dark theme). I would do that by toggling a class `demo-strong-focus-enabled` and applying this mixin like
```scss
.demo-strong-focus-enabled {
  @include mat-strong-focus-indicators();
}
```

Reply:
I added it to this CL because it's small. LMK any preference for button name & location.